packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -78,34 +80,6 @@ function renderDefaultValue(
   return defaultValue
 }
 
-const getRecommendedMaxSize = (
-  defaultAnnotationModel: ColumnModel | null | undefined,
-): number | null | undefined => {
-  return defaultAnnotationModel ? defaultAnnotationModel.maximumSize : null
-}
-
-const getCurrentMaxSize = (columnModel: ColumnModelFormData): number | null => {
-  const maxSize =
-    columnModel.maximumSize !== null
-      ? parseFloat(String(columnModel.maximumSize))
-      : null
-  const isInteger = Number.isInteger(maxSize)
-  return isInteger ? maxSize : null
-}
-
-const calculateSizeParameters = (
-  columnModel: ColumnModelFormData,
-  defaultAnnotationModel: ColumnModel | null | undefined,
-) => {
-  const currentMaxSize = getCurrentMaxSize(columnModel)
-  const recommendedSize = getRecommendedMaxSize(defaultAnnotationModel)
-  const isSmallerThanRecommendedSize =
-    recommendedSize && currentMaxSize && currentMaxSize > 0
-      ? currentMaxSize < recommendedSize
-      : false
-  return { recommendedSize, isSmallerThanRecommendedSize }
-}
-
```

Comment:
These functions were 
* moved to a utils module,
* moved to new FieldWithRecommendedMinimum component, or
* removed because we had another function that did the same thing